src/app/public/modules/list-view-grid/list-view-grid.component.html

```diff
@@ -4,15 +4,14 @@
     [data]="items | async"
     [enableMultiselect]="enableMultiselect"
     [fit]="fit"
-    [hasToolbar]="hasToolbar | async"
-    [height]="height | async"
+    [height]="gridHeight | async"
     [highlightText]="highlightSearchText ? (currentSearchText | async) : undefined"
     [rowHighlightedId]="rowHighlightedId"
     [selectedColumnIds]="selectedColumnIds | async"
     [selectedRowIds]="multiselectSelectedIds"
     [settingsKey]="settingsKey"
     [sortField]="sortField | async"
-    [width]="width | async"
+    [width]="gridWidth | async"
```

Comment:
Angular's template checker didn't like that we were putting a `number | Observable<number>` into an `async` pipe (only observables or promises are allowed). I had to use a separate public property that always returns an observable in the cases where a consumer provides only a number.